recipe/meta.yaml

```diff
@@ -7,10 +7,12 @@ package:
 
 source:
   url: https://github.com/conda/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
-  sha256: e33cf2dd0fecb887d50d7c0de13902c9e70976016b6eae336e8896892ae57d67
+  sha256: ef14dbb16179b47bef4fa6860c6467a980afa4afc7ff00521ed5c8dd358dc01a
```

Comment:
I don't know why the checksum changed.
The tag doesn't seem to have moved and the date/time of the release was before the last rebuild.
I thought, but maybe misremembered, that GitHub's tarball generation got more stable at some point in the last years..

Reply:
I don't think GitHub (and git itself) has ever offered any guarantees about the reproducibility of these auto generated tarballs.

Reply:
Yep, therefore it's okay for them to change (no guarantees). I just felt (likely wrongly so) it doesn't happen as frequently anymore.
I was surprised it changed from only 4 days ago (maybe some tooling/workflow updates/changes at GitHub?).
Then again... why don't we have generated release tarballs for `conda-build` any more? I'm opening an issue for that upstream.

Reply:
> I'm opening an issue for that upstream.

https://github.com/conda/conda-build/issues/4667